### ISSUE TITLE
[tune] JupyterNotebookReporter - Support for "/" in name

### ISSUE THREAD

**Issue by FredrikM97** [ray-project/ray#14237]
### What is the problem?
- ray - 1.2.0
- pytorch-lightning         1.2.0

Reports from JupyterNotebookReporter  show up as empty when using "/". Example:

![image](https://user-images.githubusercontent.com/16705530/108636909-70191b80-7488-11eb-8276-3634deb0ac87.png)

If I replace "loss_val" with "loss/val" the output will show up empty.

Here is a part of the output:
```
Result for _inner_2706e_00001:
  accuracy/val: 0.3412322402000427
  date: 2021-02-21_21-06-36
  done: false
  experiment_id: 0348945b48004c5f9b436b2b5302f069
  hostname: HIDDEN
  iterations_since_restore: 1
  loss/val: 59.130943298339844
  loss_train: 32.38471221923828
  node_ip: 194.47.18.213
  pid: 2594798
  time_since_restore: 58.53624129295349
  time_this_iter_s: 58.53624129295349
  time_total_s: 58.53624129295349
  timestamp: 1613937996
  timesteps_since_restore: 0
  training_iteration: 1
  trial_id: 2706e_00001
```

### Reproduction (REQUIRED)

```
TuneReportCallback(
    on="validation_end"
)
JupyterNotebookReporter(
    True
)
```

If the code snippet cannot be run by itself, the issue will be closed with "needs-repro-script".

- [ x ] I have verified my script runs in a clean environment and reproduces the issue.
- [ x ] I have verified the issue also occurs with the [latest wheels](https://docs.ray.io/en/master/installation.html).

**Comment by FredrikM97** [comment]
Checked one of the logs and it seem that the output is:
  loss/train: .nan
  loss/val: .nan

**Comment by krfricke** [comment]
Hi @FredrikM97, could you provide a self-contained reproduction script for this issue please? E.g. something I can just copy and paste in an empty file (or notebook), run, and it then shows the error. You can also link a notebook or so.

It doesn't have to be your real trainable, just something that show the same issue. Otherwise it's hard for us to debug these things, as we spend much of our time trying to rebuild training scripts.

Thanks!

**Comment by FredrikM97** [comment]
I based it upon: https://towardsdatascience.com/how-to-tune-pytorch-lightning-hyperparameters-80089a281646

The only difference is that I changed the names on self.log and did not define. metrics = {"loss": "ptl/val_loss", "acc": "ptl/val_accuracy"}
